frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.css

```diff
@@ -0,0 +1,10 @@
+
+/* Outline a blank step onBlur or form Validation */
+.ttahub-next-steps-repeater .usa-form-group--error .blank-next-step textarea,
+.usa-form-group--error .ttahub-next-steps-repeater .blank-next-step textarea {
+    border: 3px solid #b50909;
+  }
+
+.ttahub-next-steps-repeater textarea {
```

Comment:
Would remove this rule, as the textarea is set to 100% (which is what I'm seeing) and setting a min-width means that if you squish the screen down you might find the textarea overflowing outside of the screen

![Screen Shot 2022-04-28 at 2 17 20 PM](https://user-images.githubusercontent.com/3288586/165820219-0e7de224-858a-4481-b7aa-903d0e6f482e.png)